src/Tribe/Event_Repository.php

```diff
@@ -21,10 +21,59 @@ class Tribe__Tickets__Event_Repository extends Tribe__Repository__Decorator {
 	 */
 	public function __construct() {
 		$this->decorated = tribe( 'events.event-repository' );
-		$this->decorated->add_schema_entry( 'cost', array( $this, 'filter_by_cost' ) );
-		$this->decorated->add_schema_entry( 'cost_currency_symbol', array( $this, 'filter_by_cost_currency_symbol' ) );
-		$this->decorated->add_schema_entry( 'has_tickets', array( $this, 'filter_by_has_tickets' ) );
-		$this->decorated->add_schema_entry( 'has_rsvp', array( $this, 'filter_by_has_rsvp' ) );
+		$this->decorated->add_schema_entry( 'cost', [ $this, 'filter_by_cost' ] );
+		$this->decorated->add_schema_entry( 'cost_currency_symbol', [ $this, 'filter_by_cost_currency_symbol' ] );
+		$this->decorated->add_schema_entry( 'has_tickets', [ $this, 'filter_by_has_tickets' ] );
+		$this->decorated->add_schema_entry( 'has_rsvp', [ $this, 'filter_by_has_rsvp' ] );
+		$this->decorated->add_schema_entry( 'attendee', [ $this, 'filter_by_attendee' ] );
+		$this->decorated->add_schema_entry( 'attendee__not_in', [ $this, 'filter_by_attendee_not_in' ] );
+		$this->decorated->add_schema_entry( 'attendee_user', [ $this, 'filter_by_attendee_user' ] );
+
+		// This is not yet working, it needs more debugging to determine why it's not functional yet.
+		//$this->decorated->add_schema_entry( 'attendee_user__not_in', [ $this, 'filter_by_attendee_user_not_in' ] );
```

Comment:
* :no_entry_sign: No space found before comment text; expected "// $this->decorated->add_schema_entry( 'attendee_user__not_in', [ $this, 'filter_by_attendee_user_not_in' ] );" but found "//$this->decorated->add_schema_entry( 'attendee_user__not_in', [ $this, 'filter_by_attendee_user_not_in' ] );"
* :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 260020e9fdf6c865b257c91117cc2ec2</sub>